pkg/cmd/openshift-kube-apiserver/configdefault/config_default.go

```diff
@@ -0,0 +1,54 @@
+package configdefault
+
+import (
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/library-go/pkg/crypto"
+)
+
+func DefaultString(target *string, defaultVal string) {
+	if len(*target) == 0 {
```

Comment:
should you check for nil here?

Reply:
not necessary IMO. Non-nil is a programming error as this is merely a reference to a static fields. It should panic.